core/sr-primitives/src/generic/unchecked_extrinsic.rs

```diff
@@ -123,6 +130,67 @@ where
 	}
 }
 
+/// A payload that has been signed for a unchecked extrinsics.
+///
+/// Note that the payload that we sign to produce unchecked extrinsic signature
+/// is going to be different than the `SignaturePayload` - so the thing the extrinsic
+/// actually contains.
+pub struct SignedPayload<Call, Extra: SignedExtension> {
+	raw_payload: (
```

Comment:
I could not find a reason but I just wonder if you also argued/tought of making this a flat struct rather than one with a single element of a tuple? or is there a strong reason?

Reply:
Yeah, makes sense. I guess, arguably `self.raw_payload.using_encoded` is more readable than `self.0`, but indeed it feels simpler to avoid naming it.